roles/README.adoc

```diff
@@ -141,16 +141,60 @@ Consider making paths to templates internal variables if you need different temp
 ====
 
 === Supporting multiple distributions and versions
+Use Cases::
+* The role developer needs to be able to set role variables to different values
```

Comment:
Content-wise, I have no objections r/n, but please have a look at the [contribution guide](https://github.com/redhat-cop/automation-good-practices/blob/main/CONTRIBUTE.adoc); 
it states "We follow the [Asciidoc recommended practices](https://asciidoctor.org/docs/asciidoc-recommended-practices/).", and that in turn states "Don’t wrap text at a fixed column width. Instead, put each sentence on its own line, a technique called sentence per line."

Reply:
done